src/View/Helper/FormHelper.php

```diff
@@ -44,7 +44,7 @@ class FormHelper extends Helper
         'error' => '<div class="invalid-feedback">{{content}}</div>',
         'label' => '<label{{attrs}}>{{text}}{{tooltip}}</label>',
         'help' => '<small{{attrs}} class="form-text text-muted">{{content}}</small>',
-        'tooltip' => '<span data-toggle="tooltip" title="{{content}}" class="glyphicon glyphicon-info-sign"></span>',
+        'tooltip' => '<span data-toggle="tooltip" title="{{content}}" class="fas fa-info"></span>',
```

Comment:
fas => fa ?

Reply:
Nope, it's changed in FA 5, check it's docs.

Reply:
If you prefer we can use `info-circle` instead.

Reply:
Then we should probably mention FA5 requirement vs FA4.x somewhere in the docs to avoid people tripping over it.

Reply:
Maybe using "fa fas" should work for both FA 4.7 and 5.

Version 4.7 have more icons free/open - support that version would be great.

Reply:
This template as well as the defaults for `HtmlHelper::icon()` are configurable so those wanting to use FA4 can modify them when loading the helpers.